manual/introduction.md

```diff
@@ -10,14 +10,30 @@ libraryDependencies += "pl.metastack" %%  "metatime" % "%version%"  // Scala
 libraryDependencies += "pl.metastack" %%% "metatime" % "%version%"  // Scala.js
 ```
 
-## Example
-[scala type="section" value="introduction" file="Examples"]
+## Key Features
+## Type-safe DSL
+  MetaTime is a type-safe DSL which offers to prevents a lot of type errors a user might do and gives users opportunity to focus
```

Comment:
Could you rephrase this and give a couple of examples? For example, involving addition, `fromNow` etc.